### DISCUSSION TITLE
ContentButton

### DISCUSSION THREAD

**Discussion by KSemenenko** [CommunityToolkit/Maui#772] [New Feature Discussions]
do we have content button?

https://github.com/xamarin/Xamarin.Forms/issues/6872#issuecomment-511380859

**Comment by IeuanWalker** [comment]
@KSemenenko could give my NuGet a try - https://github.com/IeuanWalker/Maui.StateButton

**Comment by mhrastegari** [comment]
You can make anything clickable with `TapGestureRecognizer` so you can create something like this

```
    <Frame CornerRadius="20">
        <StackLayout Orientation="Horizontal">
            <Image/>
            <Label/>
        </StackLayout>
        <Frame.GestureRecognizers>
            <TapGestureRecognizer Command="{Binding TappedCommand}" Tapped="FrameTapped" />
        </Frame.GestureRecognizers>
    </Frame>
```

**Reply by IeuanWalker** [reply]
The issue with that is its not accessible and you also can style or animate the pressed event

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
